ceph-ansible-nightly/config/definitions/ceph-ansible-nightly.yml

```diff
@@ -132,6 +132,7 @@
       - shrink_osd
       - lvm_batch
       - add_osds
+      - add_mons
```

Comment:
from what I'm seeing, this is not going to be backported (I didn't find any `Closes:` referring to a BZ in the ceph-ansible related PR).
In that case it means it won't be run against stable-3.2, so it's not needed.
please, either remove this from stable-3.2 testing or link the related BZ in the ceph-ansible PR.

Reply:
Ok. Made the changes.